solr/core/src/test/org/apache/solr/cloud/CollectionsAPIDistributedZkTest.java

```diff
@@ -70,357 +77,193 @@
-    
     // now creating that collection should work
-    params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.CREATE.toString());
-    params.set("name", collectionName);
-    params.set("numShards", 2);
-    request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-    if (secondConfigSet) {
-      params.set("collection.configName", "conf1");
-    }
-    makeRequest(baseUrl, request);
+    response = createCollection(collectionName, configSet, 2, 1)
+        .process(cloudClient);
+    assertEquals(0, response.getStatus());
   }
   
-  private void deleteCollectionOnlyInZk() throws Exception {
-    final String baseUrl = getBaseUrl((HttpSolrClient) clients.get(0));
+  @Test
+  public void deleteCollectionOnlyInZk() throws Exception {
     String collectionName = "onlyinzk";
 
     cloudClient.getZkStateReader().getZkClient().makePath(ZkStateReader.COLLECTIONS_ZKNODE + "/" + collectionName, true);
 
-    ModifiableSolrParams params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.DELETE.toString());
-    params.set("name", collectionName);
-    QueryRequest request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-
-    makeRequest(baseUrl, request);
+    CollectionAdminResponse response = deleteCollection(collectionName)
+        .process(cloudClient);
+    assertEquals(0, response.getStatus());
 
-    assertCollectionNotExists(collectionName, 45);
+    waitForCollectionToDisappear(collectionName, cloudClient.getZkStateReader(), false, true, 45);
+    assertFalse(cloudClient.getZkStateReader().getZkClient().exists(ZkStateReader.COLLECTIONS_ZKNODE + "/" + collectionName, true));
     
     // now creating that collection should work
-    params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.CREATE.toString());
-    params.set("name", collectionName);
-    params.set("numShards", 2);
-    request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-    if (secondConfigSet) {
-      params.set("collection.configName", "conf1");
-    }
-    makeRequest(baseUrl, request);
+    response = createCollection(collectionName, configSet, 2, 1)
+        .process(cloudClient);
+    assertEquals(0, response.getStatus());
     
-    waitForRecoveriesToFinish(collectionName, false);
+    ZkStateReader zkStateReader = cloudClient.getZkStateReader();
+    waitForRecoveriesToFinish(collectionName, zkStateReader, false, true, 330);
     
-    params = new ModifiableSolrParams();
-    params.set("action", CollectionAction.DELETE.toString());
-    params.set("name", collectionName);
-    request = new QueryRequest(params);
-    request.setPath("/admin/collections");
-
-    makeRequest(baseUrl, request);
+    response = deleteCollection(collectionName)
+        .process(cloudClient);
+    assertEquals(0, response.getStatus());
   }
   
-  private void deleteCollectionWithUnloadedCore() throws Exception {
```

Comment:
Can you migrate this test as well, and ```@Ignore``` it?  At some point someone will want to fix this, I think...